packages/venia-ui/upward.yml

```diff
@@ -132,19 +132,11 @@ veniaStatic:
         inline: '.'
 
 # These are no-ops at runtime; nothing refers to these context values in the
-# rest of this file. They exist to declare that the files in the `./templates`
-# and `./static` directories are required and should be copied into the build
-# assets by the UpwardIncludePlugin. Since they are not directly mentioned
-# elsewhere in this file or any other upward.yml file in the build, the
-# UpwardIncludePlugin would fail to copy them if they were not mentioned here.
-
-# The templates directory includes partials which are only required by other
-# templates, and the UpwardIncludePlugin cannot parse templates to see them.
-veniaTemplateIncludes:
-    resolver: directory
-    directory:
-        resolver: inline
-        inline: './templates'
```

Comment:
These have not been used in PWA Studio since September 2019, after we swapped to the webpack `template.html` file. Anyone who was on the version from before then and updates since then, will have had to do this upgrade. This is just removing the files that aren't hooked up to anything.